apps/docs/content/docs/framework/artifacts/index.mdx

```diff
@@ -178,31 +207,43 @@ const versioningExample = async (callbackContext: CallbackContext) => {
 - Access latest version by default or specify version explicitly
 - Complete version history maintained
 
-## Common Use Cases
```

Comment:
You renamed this from "Common Use Cases" to "Where to start", but the code snippets that follow after the cards are a bit confusing. Were they supposed to be common use cases? Can you add context to them, so the user can know what the "File Upload and Processing" and "Caching Expensive Operations" are for?